diff --git a/webrtc/base/base.gyp b/webrtc/base/base.gyp
index 080d203..ca6a4bc 100644
--- a/webrtc/base/base.gyp
+++ b/webrtc/base/base.gyp
@@ -232,6 +232,7 @@
         'FEATURE_ENABLE_SSL',
         'SSL_USE_OPENSSL',
         'HAVE_OPENSSL_SSL_H',
+        'NO_MAIN_THREAD_WRAPPING'
       ],
       'sources': [
         'applefilesystem.mm',
@@ -395,14 +396,6 @@
               ],
             }],
           ],
-          'defines': [
-            'NO_MAIN_THREAD_WRAPPING',
-          ],
-          'direct_dependent_settings': {
-            'defines': [
-              'NO_MAIN_THREAD_WRAPPING',
-            ],
-          },
         }, {
           'sources': [
             'callback.h',
